src/component/mxgraph/config/StyleConfigurator.ts

```diff
@@ -298,6 +298,10 @@ export default class StyleConfigurator {
         if (markers.length > 0) {
           styleValues.set(StyleIdentifier.BPMN_STYLE_MARKERS, markers.join(','));
         }
+      } else if (bpmnElement.kind === ShapeBpmnElementKind.POOL) {
+        // mxConstants.STYLE_HORIZONTAL is for the label
+        // In BPMN, isHorizontal is for the Pool
+        styleValues.set(mxConstants.STYLE_HORIZONTAL, bpmnCell.isHorizontal ? '0' : '1');
```

Comment:
:warning: I would prefer true/false string as it is more readable/understandable
This is what we did when we had a dedicated style for the `isExpanded` (prior we manage it as the rest of activity marker)